### PR TITLE
Light dark theme

### PR DESCRIPTION
**Pull request by kristoferjoseph**
This depends on the next version of Enhance styles

### CODE REVIEW

**Comment by kristoferjoseph** [comment]
Enhance styles 6 and the plugin 4 were merged into plugin Enhance so this should be good to go.

I moved focus handling and invalid state into the root css so all inputs get the update.

File: packages/page-container/PageContainer.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ export default function PageContainer({ html }) {
     return html`
 <style>
   :host {
-    color: var(--dark);
-    background-color: var(--light)
+    color: var(--fore);
+    background-color: var(--back)
```
Comment (macdonst) [inline packages/page-container/PageContainer.js:7]: Missing semi-colon

File: packages/link-element/LinkElement.js [hunk 2]
```diff
@@ -4,14 +4,15 @@ export default function LinkElement({ html, state }) {
     return html`
 <style>
   :host {
-    color: var(--primary-500);
+    color: var(--accent);
```
Comment (macdonst) [inline packages/link-element/LinkElement.js]: aXe and Lighthouse report contrast issues with this color in light and dark mode.

File: packages/text-input/TextInput.js [hunk 3]
```diff
@@ -30,20 +30,17 @@ export default function TextInput({ html, state }) {
...
   :host label input:focus {
-    outline: none;
-    border-color: var(--grey-700);
-    transition: border-color 0.15s ease-in;
+    transition: outline 0.15s ease-in;
   }
   :host .errors {
-    color: var(--error-500);
-    border-color: var(--error-500);
+    color: var(--error);
```
Comment (macdonst) [inline packages/text-input/TextInput.js]: aXe and Lighthouse report contrast issues with this color in light and dark mode.

File: packages/text-input/TextInput.js [hunk 4]
```diff
@@ -30,20 +30,17 @@ export default function TextInput({ html, state }) {
...
   }
   :host p:not(.errors) {
-    color: var(--muted)
+    color: var(--grey-600)
```
Comment (macdonst) [inline packages/text-input/TextInput.js]: aXe and Lighthouse report contrast issues with this color in dark mode.

**Review by macdonst — APPROVED** [review]
LGTM, ran into some issues with color contrast but you'd know better than I ever they are valid issues. Just flagging them for now.

File: packages/check-box/CheckBox.js [hunk 5]
```diff
@@ -29,36 +21,67 @@ export default function CheckBox({ html, state }) {
...
+
+      :host label input[type="checkbox"]::before {
+        content: "";
+        aspect-ratio: 1/1;
+        width: 100%;
+        transform: scale(0);
+        transition: 120ms transform ease-in-out;
+        box-shadow: inset 1rem 1rem var(--accent);
+        background-color: CanvasText;
+        clip-path: polygon(14% 44%, 0 65%, 50% 100%, 100% 16%, 80% 0%, 43% 62%);
```
Comment (colepeters) [inline packages/check-box/CheckBox.js:51]: What's all this about? 😄

File: packages/check-box/CheckBox.js [hunk 6]
```diff
@@ -29,36 +21,67 @@ export default function CheckBox({ html, state }) {
...
+      "
+    >
       <input
-        class="p-2 flex-grow radius0 border-solid mbe-2 mie-4 border1 select-none ${errors ? 'errors' : ''}"
+        class="
+          border-current
+          border-solid
+          border1
+          select-none
+          text1
```
Comment (colepeters) [inline packages/check-box/CheckBox.js]: Wondering about `text1` here — does it do anything, being on the `input`?

**Review by colepeters — COMMENTED** [review]
Just a couple questions!

File: packages/check-box/CheckBox.js (hunk 5 above)
Comment (kristoferjoseph) [reply packages/check-box/CheckBox.js:51]: I ran into an issue with the focus ring on the checkbox not conforming to the border radius. The fix was to make the checkbox with CSS. Downside is it all had to be redrawn, even the checkmark. CanvasText is for windows only.

File: packages/text-input/TextInput.js (hunk 3 above)
Comment (kristoferjoseph) [reply packages/text-input/TextInput.js]: These values are directly from the Apple human interface guides. I will pick some higher contrast colors.

File: packages/text-input/TextInput.js (hunk 4 above)
Comment (kristoferjoseph) [reply packages/text-input/TextInput.js]: Ha! I took these values directly from the Apple human interface guides. I will pick some higher contrast colors.

File: packages/check-box/CheckBox.js (hunk 5 above)
Comment (colepeters) [reply packages/check-box/CheckBox.js:51]: 🤯

**Review by macdonst — APPROVED** [review]

**Pull request merged**